cvat-core/src/server-proxy.js

```diff
@@ -619,19 +624,91 @@
                     throw generateError(errorData);
                 }
 
-                onUpdate('The data are being uploaded to the server..');
-                try {
-                    await Axios.post(`${backendAPI}/tasks/${response.data.id}/data`, taskData, {
-                        proxy: config.proxy,
-                    });
-                } catch (errorData) {
-                    try {
-                        await deleteTask(response.data.id);
-                    } catch (_) {
-                        // ignore
+                onUpdate('The data are being uploaded to the server 0%');
+
+                async function chunkUpload(file) {
+                    const chunkSize = 1024 * 1024 * 100; // 100 mb
+                    const totalChunks = Math.ceil(file.size / chunkSize);
+                    let currentChunkNumber = 1;
+                    async function upload() {
+                        const prevChunkNumber = currentChunkNumber - 1;
+                        const fileChunk = file.slice(prevChunkNumber * chunkSize, Math.min(prevChunkNumber * chunkSize + chunkSize, file.size));
+                        taskData.set('client_files[0]', fileChunk);
+                        taskData.set('chunk_file_name', file.name);
+                        taskData.set('end_of_upload', currentChunkNumber >= totalChunks);
+                        taskData.set('use_chunk_upload', true);
+                        try {
+                            await Axios.post(`${backendAPI}/tasks/${response.data.id}/data`, taskData, {
+                                proxy: config.proxy,
+                            });
+                            onUpdate(`The data are being uploaded to the server ${Math.round((currentChunkNumber / totalChunks) * 100)}%`);
+                            currentChunkNumber++;
+                            if (currentChunkNumber <= totalChunks) await upload();
```

Comment:
Do not need try inside if we wrap the function call (chunkUpload and bulkUpload).

Reply:
Done